### PR TITLE
Clean up registry mirror docs

### PR DESCRIPTION
**Pull request by abhinavmpandey08**
*Issue #, if available:*

*Description of changes:*
Clean up registry mirror docs, and make it more consistent with the cluster spec

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere/pull/540#" title="Author self-approved">abhinavmpandey08</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[docs/OWNERS](https://github.com/aws/eks-anywhere/blob/main/docs/OWNERS)~~ [abhinavmpandey08]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: docs/content/en/docs/reference/clusterspec/registrymirror.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+```
+## Docker configurations
+It is necessary to add the private registry's CA Certificate
+to the list of CA certificates on the admin machine if your registry uses self-signed certificates.
+
+For [Linux](https://docs.docker.com/engine/security/certificates/), you can place your certificate here: `/etc/docker/certs.d/<private-registry-endpoint>/ca.crt`
+
+For [Mac](https://docs.docker.com/desktop/mac/#add-tls-certificates), you can follow this guide to add the certificate to your keychain: https://docs.docker.com/desktop/mac/#add-tls-certificates
+
+>>>NOTE: You may need to restart Docker after adding the certificates.
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/clusterspec/registrymirror.md:71]:
```suggestion
% alert title="Note" color="primary" %
You may need to restart Docker after adding the certificates.
% /alert %
```
You an use this style for marking notes.

**Review by chrisnegus — COMMENTED** [review]
Just one small suggestion for formatting the note.

File: docs/content/en/docs/reference/clusterspec/registrymirror.md (hunk 1 above)
Comment (abhinavmpandey08) [reply docs/content/en/docs/reference/clusterspec/registrymirror.md]: done

**Pull request merged**